<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>300</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Choose Filter Option</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCountryBids">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter By Country - Bidders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCategory">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter By Category</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDateRange">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter By Date Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKeyword">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter By Keyword</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCountryPosters">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter By Country - Posters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCountryWinners">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter By Country - Winners</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
